### PR TITLE
User can accept requests

### PR DESCRIPTION
**Pull request by kaylawoodbury**
### User Story
```
As a user
In order to accept requests
I would like an interface to accept or reject incoming requests
```
## Changes proposed in this pull request:
* Adds tests and fixture for user can accept or reject tests
* Adds button and new page for reviewing all pending and accepted requests for the active trip
* Adds accept and reject buttons and functionality for each request
* Displays accepted requests with check boxes
* Adds styling 

## What I have learned working on this feature:
* how to add icons with react native
* how to implement a switch case statement for a FlatList map function 

## Packages/Gems added
- react-native-elements
- react-native-vector-icons

## Screenshots (Client)
Screenshot 2020-04-17 at 21 32 23
Screenshot 2020-04-17 at 21 32 09

## Lo-Fi's (Client)
Pasted Image at 2020_04_11_01_50 pm

### CODE REVIEW

File: modules/tripActions.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import axios from "axios";
-import { GET_TRIP_LIST } from "../state/actions/actionTypes";
+import { GET_TRIP_LIST, GET_TRIP_DETAILS, GET_TRIP_REQUEST_DETAILS, PONG_STATUS } from "../state/actions/actionTypes";
+
+let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
 
 const fetchTrips = async (dispatch) => {
-  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  // let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
   let response = await axios.get("https://co-ping.herokuapp.com/pings", {
```
Comment (emtalen) [inline modules/tripActions.js]: remove comments

File: screens/RequestList.jsx [hunk 2]
```diff
@@ -0,0 +1,215 @@
+import React, { useEffect } from "react";
+import { LinearGradient } from "expo-linear-gradient";
+import { View, StyleSheet, Text, FlatList, TouchableHighlight } from "react-native";
+import { useSelector, useDispatch } from "react-redux";
+import { getInformation, acceptRequest, rejectRequest } from "../modules/tripActions"
+import { CheckBox, Icon } from 'react-native-elements'
+
+
```
Comment (emtalen) [inline screens/RequestList.jsx:17]: an extra line here

**Review by emtalen — COMMENTED** [review]
Looks really awesome 💯

File: modules/tripActions.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import axios from "axios";
-import { GET_TRIP_LIST } from "../state/actions/actionTypes";
+import { GET_TRIP_LIST, GET_TRIP_DETAILS, GET_TRIP_REQUEST_DETAILS, PONG_STATUS } from "../state/actions/actionTypes";
+
+let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
 
 const fetchTrips = async (dispatch) => {
-  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  // let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
```
Comment (pgaunitz) [inline modules/tripActions.js]: remove commented code

**Review by pgaunitz — APPROVED** [review]

**Review by emtalen — APPROVED** [review]
Winners!!

**Review by pgaunitz — APPROVED** [review]
Fantastic

**Pull request merged**